Packs/Active_Directory_Query/Integrations/Active_Directory_Query/README.md

```diff
@@ -1016,6 +1107,10 @@ Used in the IAM premium pack.
 #### Base Command
 
 `iam-disable-user`
+
+##### Required Permissions
+Requires `Create, delete, and manage user accounts` permissions.
```

Comment:
should be changed to this 
`Requires Read userAccountControl and write userAccountControl permissions.`
Similar to `ad-disable-account`